### ISSUE TITLE
error on Windows 10 OS: Environment variable not found: DATABASE_URL.

### ISSUE THREAD

**Issue by pegaltier** [backtestjs/command-line#1]
on Windows 10 OS
npm run align-db
error: Environment variable not found: DATABASE_URL.

<img width="592" alt="image" src="https://github.com/user-attachments/assets/2f08f01d-ccfd-4010-a8af-b098ece3bfcb" />

**Comment by pegaltier** [comment]
Solution: 

"align-db": "set DATABASE_URL=file:D:\\path\\command-line\\db\\backtest.db && npx prisma validate && npx prisma generate && npx prisma db push",

can be added to the doc; or better we can use a solution like cross env

**Comment by mrddter** [comment]
Sure, tomorrow I’ll update docs with env parameters. Thanks Pegaltier

**Comment by mrddter** [comment]
Here I am again, again, thank you @pegaltier.

TL;DR Besides passing the parameter as you mentioned, you can also (only) create a .env file.

I have updated the README.md for the projects [command-line](https://github.com/backtestjs/command-line), [framework](https://github.com/backtestjs/framework), and [quick-start](https://github.com/backtestjs/quick-start) for those who want to use it from CLI or as dependencies.

Getting back on track, for passing environment variables, you can do it as you mentioned or - I suggest - use a .env file ([Node.js 20 supports .env files](https://nodejs.org/en/learn/command-line/how-to-read-environment-variables-from-nodejs)). 

If you have any other suggestions or feedback, let me know ❤️
